README.md

```diff
@@ -13,6 +13,8 @@ A few backwards incompatible API changes were included with version 2.0.0.
 
 * The `chatJoin` and `chatLeave` events were changed to return the channel name instead of the id. All debug logs pertaining chat channels will now mention chat channel instead of IDs.
 
+* The `disabledGroups` argument in the `matchmakingStatsData` event has been removed.
```

Comment:
Don't you mean `wait_times_by_group`? You should also list the event name changes.